rqt_gui/package.xml

```diff
@@ -15,6 +17,9 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <build_depend version_gte="0.3.0">qt_gui</build_depend>
-  <run_depend>catkin</run_depend>
-  <run_depend version_gte="0.3.0">qt_gui</run_depend>
+
+  <exec_depend>catkin</exec_depend>
```

Comment:
I'm surprised `catkin` was a `run_depend`. Maybe it's supposed to be `buildtool_export_depend`?

Reply:
Looks like it was added in 2012 in 6966d37 shortly after the package was catkinized. I don't really follow the reasoning in the commit message.

Reply:
This should certainly be removed. I don't see any reason why it should be a run dependency.